packages/url_launcher/url_launcher_web/lib/url_launcher_web.dart

```diff
@@ -4,7 +4,7 @@ import 'dart:html' as html;
 import 'package:flutter_web_plugins/flutter_web_plugins.dart';
 import 'package:meta/meta.dart';
 import 'package:url_launcher_platform_interface/url_launcher_platform_interface.dart';
-import 'navigator.dart' as navigator;
+import 'src/navigator.dart' as navigator;
```

Comment:
(No need for the `as navigator` anymore)

Reply:
If I remove the prefix declaration, the analyzer throws:

>  info • Unused import: 'src/navigator.dart' • lib/url_launcher_web.dart:7:8 • unused_import  
error • Undefined name 'navigator' • lib/url_launcher_web.dart:25:20 • undefined_identifier

Reply:
Ah, you get the unprefixed getter (you could call `standalone` directly. You're right, this is better :)